analysis/import/GitLogParser/src/main/kotlin/de/maibornwolff/codecharta/importer/gitlogparser/GitLogParser.kt

```diff
@@ -134,58 +110,25 @@ class GitLogParser(
     }
 
     private fun createProjectFromLog(
-        pathToLog: File,
-        pathToNameTree: File,
+        gitLogFile: File,
+        gitLsFile: File,
         parserStrategy: LogParserStrategy,
         metricsFactory: MetricsFactory,
         containsAuthors: Boolean,
         silent: Boolean = false
     ): Project {
-        val namesInProject = readFileNameListFile(pathToNameTree)
-        val encoding = guessEncoding(pathToLog) ?: "UTF-8"
+        val namesInProject = readFileNameListFile(gitLsFile)
```

Comment:
I would always try to not use a abbreviation/acronym in variable names. By that it is mostly hard for new developers to understand the purpose of the variable.

Reply:
I think I remember talking about this with @Hall-Ma, we a lot of back and forth about this name too. Something like gitFiles or trackedFiles(List) would make the datatype hard to read/guess at a glance. We settled on this since we thought that the "ls" command is well known enough and can be inferred from both the docs and the command description.
Or would something like gitFileNameListFile be alright with you?

Reply:
I didn't get the point that you mean the list command by `Ls`. :-)
I admit it is a bit tricky to combine the type "File" with the variable name. So I would just sugggest something like `listOfRepositoryFiles`. But this is nit picking. It is fine for me to keep it for now.